### PR TITLE
Feat/star

### PR DESCRIPTION
**Pull request by ivyhu630**
Minor styling on star component & display of two review blocks. Buttons currently do not work yet.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "axios": "^0.26.1",
     "dotenv": "^16.0.0",
     "express": "^4.17.3",
+    "moment": "^2.29.2",
```
Comment (floridamaniac) [inline package.json:25]: heads up @theVikingMan @ryand8008 we'll need to `npm i` after pulling this one

File: client/src/contexts/ProductReview.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+export function ReviewProvider({ children }) {
+  const productId = useContext(ProductIDContext);
+  const [reviews, setReviews] = useState({});
+
+  useEffect(() => {
+    axios({
+      method: 'get',
+      url: 'http://localhost:3000/reviews',
```
Comment (floridamaniac) [inline client/src/contexts/ProductReview.jsx:21]: You should be able to use `/reviews` without the localhost here. Will save us from having to hunt it down once we start deploying to production

File: client/src/contexts/ProductReview.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect, useContext } from 'react';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+import { ProductIDContext } from './ProductIDContext';
+
+const ReviewContext = React.createContext();
+ReviewContext.displayName = 'ReviewData';
+
+// custome hook to return useContext
+export function useReviews() {
```
Comment (floridamaniac) [inline client/src/contexts/ProductReview.jsx:10]: fab

File: client/src/components/Ratings/ReviewList.jsx [hunk 4]
```diff
@@ -1,21 +1,50 @@
...
+  }
   return (
-    <ReviewProvider>
-      <RatingContainer>
-        ReviewList
-        <ReviewTile />
-      </RatingContainer>
-    </ReviewProvider>
+    <ReviewContainer>
+      {reviewCollection}
```
Comment (floridamaniac) [inline client/src/components/Ratings/ReviewList.jsx:22]: Oh this is a cool way to do conditional rendering.

File: client/dist/index.html [hunk 5]
```diff
@@ -5,7 +5,6 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Adventure Time</title>
     <link rel="stylesheet" href="styles.css"></link>
-    <!-- // using W3 stylesheet for progressbar -->
     <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
```
Comment (floridamaniac) [inline client/dist/index.html:8]: Would be fab to get the relevant styles set up ourselves so we're not pulling in another file, but it's non-breaking

**Review by floridamaniac — APPROVED** [review]
It's looking goooooood

File: client/src/contexts/ProductReview.jsx (hunk 2 above)
Comment (ivyhu630) [reply client/src/contexts/ProductReview.jsx:21]: Will update

File: client/dist/index.html (hunk 5 above)
Comment (ivyhu630) [reply client/dist/index.html:8]: Will do. That is on my to-do list. Alex had a similar comment earlier.

**Pull request merged**